search.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+# for details. All rights reserved. Use of this source code is governed by a
+# BSD-style license that can be found in the LICENSE file.
+
+runtime: custom
+env: flex
+service: search
+env_variables:
+  # Needs to be in sync with lib/shared/configuration.dart
+  DUCENE_DIR: '/ducene'
+
+resources:
```

Comment:
https://cloud.google.com/compute/docs/machine-types#standard_machine_types

FYI – I think you get a machine type that is at least the requirements you put here.

...so I believe this would bump us up to a 2-cpu box.

So drop ram to 3.5?